services/QuillLMS/client/app/bundles/PremiumHub/containers/diagnosticGrowthReports/index.tsx

```diff
@@ -58,10 +60,29 @@ export const DiagnosticGrowthReportsContainer = ({
 }) => {
 
   const [activeTab, setActiveTab] = React.useState<string>(OVERVIEW)
-  const [selectedDiagnosticId, setSelectedDiagnosticId] = React.useState<number>(null)
+  const [selectedDiagnosticType, setSelectedDiagnosticType] = React.useState<DropdownObjectInterface>(null)
   const [selectedGroupByValue, setSelectedGroupByValue] = React.useState<DropdownObjectInterface>(null)
   const [noDiagnosticDataAvailable, setNoDiagnosticDataAvailable] = React.useState<boolean>(!!passedData)
 
+  React.useEffect(() => {
+    // this is for testing purposes; this value will always be null in a non-testing environment
+    if (!passedData) {
+      getFilterSelections(activeTab)
+    }
+  }, [])
+
+  React.useEffect(() => {
+    if (!passedData) {
+      getFilterSelections(activeTab)
+    }
+  }, [activeTab])
```

Comment:
You shouldn't need both of these -- the one with `[activeTab]` should run on the initial render as well as when the value changes.

Reply:
oops-- meant to take that out, thanks for catching!